shell/lib/db.js

```diff
@@ -326,7 +326,7 @@ if (Meteor.isServer) {
 
     if (this.userId) {
       return Meteor.users.find({_id: this.userId},
-          {fields: {signupKey: 1, isAdmin: 1, expires: 1}});
+          {fields: {signupKey: 1, isAdmin: 1, expires: 1, quota: 1, storageUsage: 1}});
```

Comment:
We've discussed that we eventually plan to allow user accounts to encompass multiple identities. I imagine that entries in `Meteor.users()` will continue to correspond to identities, and we'll have a new `Accounts` table to collect identities. That `Accounts` table is where the quota data should ultimately end up, I think. So I guess that putting the quota data here is just a temporary solution?